### PR TITLE
WEB-1701 Unable to apply Vouchers in Staging

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** Unable to apply Vouchers in Staging

**JIRA:**  https://liv-it.atlassian.net/browse/WEB-1701

**Merge deadline:** N/A

**Testing instructions:**

Steps to reproduce:
1. Create License (mine:Mk9qXoero4hWeq1MvdRWxwYexMVs) 
2. Apply new license in a new course (Teacher: elvana+119@labster.com) 
3. Create vouchers 
4. Login as student (Student: elvana+50@labster.com) 
5. Enter Voucher (Voucher:GF8JEGN2W2 or 8M4GFCBOWI)

**Reviewers:**
- [x] @TamoshaytisV  
- [x] @leha2000 

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [x] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by TamoshaytisV** [comment]
Can you please add comment that this fix is voucher system related?

File: cms/djangoapps/contentstore/views/course.py [hunk 1]
```diff
@@ -1249,6 +1250,15 @@ def advanced_settings_handler(request, course_key_string):
...
                         modulestore().update_item(course_module, request.user.id)
+                        # Throw away old versions of CourseOverview and CourseOverviewTab - added by labster
+                        # as they might contain stale data then create new one.
+                        try:
+                            course_overview = CourseOverview.objects.get(id=course_key)
+                            CourseOverviewTab.objects.filter(course_overview=course_overview).delete()
+                            course_overview.delete()
+                        except CourseOverview.DoesNotExist:
+                            pass
+                        CourseOverview.load_from_module_store(course_key)
```
Comment (lukyfebrianto) [inline cms/djangoapps/contentstore/views/course.py]: Delete old data of CourseOverview and CourseOverviewTab then
Create new one CourseOverview from data course module that has been updated so when user try to activate a voucher will process with up to date Data/settings(ex. invitation_only).

Comment (lukyfebrianto) [reply cms/djangoapps/contentstore/views/course.py]: ^^ @TamoshaytisV

**Review by TamoshaytisV — APPROVED** [review]
👍

**Review by leha2000 — APPROVED** [review]

**Pull request merged**